src/odemis/gui/model/__init__.py

```diff
@@ -230,6 +230,8 @@ def __init__(self, microscope):
         self.streak_unit = None  # streak unit of the streak camera
         self.streak_delay = None  # delay generator of the streak camera
         self.streak_lens = None  # input optics in front of the streak camera
+        self.tc_od_filter = None
```

Comment:
Could you also update _ROLE_TO_ATTR, so that they actually contain the right component when it's present?